csrc/runtime/executor.cpp

```diff
@@ -304,6 +305,9 @@ void KernelExecutor::compile(
   NVF_ERROR(
       !fusion->outputs().empty(), "No output found for this kernel, aborting.");
 
+  createKernelId(
```

Comment:
Remove https://github.com/NVIDIA/Fuser/blob/d62322174e7439ab4623407b6eab3e13667b0921/csrc/runtime/executor.cpp#L421-L422? I don't think there's value to create twice.

Reply:
I just moved it here.

I also am not sure if this should be a stateful operation. It seems just returning a string seems sufficient, but didn't bother to do a further cleanup.